setup.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name='quickcache',
+    version='0.0.1',
+    description='caching has never been easier',
+    author='Dimagi',
+    author_email='dev@dimagi.com',
+    url='https://github.com/dimagi/quickcache',
+    packages=['quickcache'],
+    test_suite='test_quickcache',
+    install_requires=[],
```

Comment:
Nope, it's always used mocks for the actual cache backend, and I removed some unnecessary dependencies on django libs in the test.

`django` is now only imported in `quickcache_django.py`, so if you steer clear of that (which our tests do), there's no external dependency.